ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/ServerUtil.java

```diff
@@ -51,8 +51,15 @@ public static HostInfo parseHostInfo(final String applicationServerId) {
     if (applicationServerId == null || applicationServerId.trim().isEmpty()) {
       return StreamsMetadataState.UNKNOWN_HOST;
     }
-    final String host = getHost(applicationServerId);
-    final Integer port = getPort(applicationServerId);
+
+    // the getHost/getPort utilities don't handle URLs that end with `/` - since this issue
+    // has come up multiple times for ksqlDB we special case it here
```

Comment:
I don't think the comment adds anything to the code.

How about removing and maybe  reference the issue in the test?

Reply:
fair enough